updateFeatureFlag/main.go

```diff
@@ -72,8 +72,8 @@ func updateFeatureFlag(flagId string, updateFeatureFlagRequest utils.UpdateFeatu
 
 	//throw the response on conditional check failed exception
 	if err != nil {
-		if aerr, ok := err.(awserr.Error); ok {
-			if aerr.Code() == dynamodb.ErrCodeConditionalCheckFailedException {
+		if awsErr, ok := err.(awserr.Error); ok {
+			if awsErr.Code() == dynamodb.ErrCodeConditionalCheckFailedException {
```

Comment:
please look at the comment provided in pr #38

Reply:
In this line 
```go
if awsErr, ok := err.(awserr.Error); ok {
```
we are checking if the err variable is of type awserr.Error.

Reply:
@RitikJaiswal75 Resolving this as discussed